src/libYARP_dev/src/devices/Rangefinder2DWrapper/Rangefinder2DWrapper.cpp

```diff
@@ -511,6 +512,27 @@ bool Rangefinder2DWrapper::open(yarp::os::Searchable &config)
         return false;
     }
 
+    if(config.check("subdevice"))
+    {
+        Property       p;
+        PolyDriverList driverlist;
+        p.fromString(config.toString(), false);
+        p.put("device", config.find("subdevice").asString());
+
+        if(!driver.open(p) || !driver.isValid())
+        {
+            yError() << "failed to open subdevice.. check params";
```

Comment:
Adding a "Rangefinder2DWrapper: " prefix to the error message helps a lot when reading logs.